charts/keycloak/templates/service-http.yaml

```diff
@@ -19,6 +19,12 @@ spec:
   {{- if and (eq "LoadBalancer" .Values.service.type) .Values.service.loadBalancerIP }}
   loadBalancerIP: {{ .Values.service.loadBalancerIP }}
   {{- end }}
+  {{- if .Values.service.sessionAffinity }}
+  sessionAffinity: {{ .Values.service.sessionAffinity }}
+  {{- if .Values.service.sessionAffinityConfig }}
+  sessionAffinityConfig: {{- toYaml .Values.service.sessionAffinityConfig | nindent 4 }}
+  {{- end }}
```

Comment:
```suggestion
  {{- with .Values.service.sessionAffinityConfig }}
  sessionAffinityConfig: 
  	{{- toYaml . | nindent 4 }}
  {{- end }}
```

Reply:
I use here `nindent` instead of `indent` so new line will be added with this function. If you want to follow suggested formatting I will change it to `indent`

Reply:
updated